CodeGeneration/Sources/SyntaxSupport/GrammarGenerator.swift

```diff
@@ -0,0 +1,82 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+
+/// Generates grammar doc comments for syntax nodes.
+struct GrammarGenerator {
+  let lookThroughCollections: Bool
+
+  private func grammar(for tokenChoice: TokenChoice) -> String {
+    switch tokenChoice {
+    case .keyword(text: let text):
+      return "`'\(text)'`"
```

Comment:
I feel having quotation marks within the code voice makes it a bit ambiguous what the token actually is.

Reply:
My idea here was to optimize this for rendering in docc and I found that rendering the token name in code voice reads a little nicer in docc. Would you prefer to just use 

```swift
return "'\(text)'"
```

Reply:
The thing is that we use docc links for links to other syntactic categories, which are always monospaced. 

The documentation for [StructDeclSyntax](https://swiftpackageindex.com/apple/swift-syntax/main/documentation/swiftsyntax/structdeclsyntax#Children) shows all kinds of references. Which fonts would you use here?